geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/util/SpatialiteUtilities.java

```diff
@@ -43,12 +43,74 @@
      * Array of fields that will be ingored in attributes handling.
      */
     public static String[] IGNORED_FIELDS = {SPATIALTABLE_ID_FIELD, "PK_UID", "_id"};
-
+     
     /**
      * Name/path separator for spatialite table names.
      */
     public static final String UNIQUENAME_SEPARATOR = "#"; //$NON-NLS-1$
 
+    /**
+     * Active Connection to a  Spatialite Database
+    * <p/>
+     * <p>Indented for general use for Transformation etc.
+     */
+    private static Database dbSpatialite=null;
```

Comment:
I am quite sure I don't want these to be STATIC. Using complex objects as static in Android is way to risky for memory leaks. This NEEDS to be changed most probably. Same below for memory db.